src/Rectangle.java

```diff
@@ -0,0 +1,19 @@
+
+public class Rectangle {
+    public int lengthOfObject;
+    public int widthOfObject;
```

Comment:
Ми повині робити поля приватними, це є базис інкапсуліції та і імя можна простіше зробити
```suggestion
    private int length;
    private int width;
```